eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -410,20 +411,30 @@ describe("questionnaire", () => {
         {
           bodyText: "I am note",
           publishStatus: "Unpublished",
-          questionnaireTitle: "Questionnaire",
+          questionnaireTitle: "Questionnaire (Version 1)",
           user: {
             email: "eq-team@ons.gov.uk",
           },
         },
         {
           bodyText: null,
           publishStatus: "Questionnaire created",
-          questionnaireTitle: "Questionnaire",
+          questionnaireTitle: "Questionnaire (Version 1)",
           user: {
             email: "eq-team@ons.gov.uk",
           },
         },
       ]);
     });
   });
+
+  describe("versioning", () => {
+    ctx = buildContext(null);
+
+    it("should initialise new questionnaires at version 1", () => {
+      expect(ctx.questionnaire.surveyVersion).toEqual("1");
+    });
+
+    it("can increment the version number of a questionnaire if a published survey is modified and becomes unpublished", () => {});
```

Comment:
This test is not doing anything

Reply:
Ah I forgot about that, haha. Removed and replaced!